source/form.html

```diff
@@ -0,0 +1,129 @@
+    <a class="logo" href="index.html"><img class="logo__image" src="#" alt="Текстовый логотип Кэт Энерджи."></a>
+    <button class="page-header__button-close button-close" type="button" aria-label="Закрыть"></button>
+  </div>
+  <nav class="site-navigation">
+    <ul class="site-navigation__list site-nav-list">
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="catalog.html">Каталог продукции</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="index.html">Главная</a></li>
+      <li class="site-nav-list__item"><a class="site-nav-list__link" href="form.html">Подбор программы</a></li>
+    </ul>
+  </nav>
+</header>
+<main class="page-main">
+  <h1 class="page-main__headline main-headline">Подбор программы</h1>
+  <p class="page-main__text">Заполните анкету и мы подберем программу питания для вашего кота</p>
+  <form class="page-main__form form" action="https://echo.htmlacademy.ru" method="post">
+    <fieldset class="form__fieldset">
+      <ul class="form__list">
+        <li class="form__item">
+          <label class="form__label" for="cat-name">Имя:*</label>
+          <input class="form__input" id="cat-name" name="cat-name" placeholder="Барсик" type="text" required>
+        </li>
+        <li class="form__item">
+          <label class="form__label" for="cat-weight">Вес (кг):*</label>
+          <input class="form__input" id="cat-weight" name="cat-weight" placeholder="7" type="number" required>
+        </li>
+        <li class="form__item">
+          <label class="form__label" for="cat-age">Возраст (лет):</label>
+          <input class="form__input" id="cat-age" name="cat-age" placeholder="7" type="number">
+        </li>
+      </ul>
+    </fieldset>
+    <fieldset class="form__fieldset">
+      <ul class="form__list">
+        <li class="form__item">
+          <input class="form__radio" id="weight-loss" name="about-weight" type="radio" value="weight-loss" checked>
+          <label class="form__label" for="weight-loss">Похудение</label>
+        </li>
+        <li class="form__item">
+          <input class="form__radio" id="weight-gain" name="about-weight" type="radio" value="weight-gain">
+          <label class="form__label" for="weight-gain">Набор массы</label>
+        </li>
+        <li class="form__item">
+          <input class="form__radio" id="idk-help-me" name="about-weight" type="radio" value="i-dont-know">
+          <label class="form__label" for="idk-help-me">Не знаю (Нужен ваш совет)</label>
+        </li>
+      </ul>
+    </fieldset>
+    <fieldset class="form__fieldset">
+      <h2 class="form__headline thematic-headline">Контактные данные (владельца кота)</h2>
+      <ul class="form__list">
+        <li class="form__item">
+          <label class="form__label" for="email">E-mail:*</label>
+          <input class="form__input form__input--letter" id="email" name="email" type="email" required>
```

Comment:
```suggestion
          <input class="form__input form__input--svg-icon" id="email" name="email" type="email" required>
```
или `form__input--with-svg` потому что они для обоих инпутов вставятся одинаковым макаром. Когда будешь делать увидишь